commands/root.go

```diff
@@ -19,13 +19,13 @@ import (
 	"fmt"
 	"log"
 
+	"github.com/spf13/cobra"
+
 	clusterDir "github.com/hazelcast/hazelcast-commandline-client/commands/cluster"
 	mapDir "github.com/hazelcast/hazelcast-commandline-client/commands/types/map"
```

Comment:
This is not your code, but can you change `clusterDir` to something like `clustercmd` and `mapDir` to `mapcmd`?

Reply:
Sorry, handled this but somehow reverted I guess, will do